guides/k8s-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,323 @@
+# How to deploy microservices to Kubernetes
+
+## The problem
+
+You've created a brand new microservices system using [Lagom](http://www.lagomframework.com/). After evaluating all of 
+your deployment options, you've chosen to deploy to [Kubernetes](https://kubernetes.io/) to leverage the facilities it 
+provides for automated deployment, scaling, and management of containerized applications. This raises a problem though: 
+since Lagom is based on [Play](https://www.playframework.com/) and [Akka](http://akka.io/) and leverages Lagom's 
+[Persistent Entity API](https://www.lagomframework.com/documentation/1.3.x/java/PersistentEntityCassandra.html), 
+Akka cluster is required. Lagom applications also need the ability to locate other services running in its environment.
+Finally, running an application on Kubernetes requires containerization. How exactly are these complex systems 
+deployed to Kubernetes?
+
+## The solution
+
+In this guide, the steps required to deploy a Lagom microservices system to your local Kubernetes cluster, by
```

Comment:
And, I actually didn't see anything specific to BlueMix, so I suspect you are still working on them.